packages/expo-updates/android/src/main/java/expo/modules/updates/UpdatesDevLauncherController.kt

```diff
@@ -244,13 +244,8 @@ class UpdatesDevLauncherController(
           databaseHolder.releaseDatabase()
           this@UpdatesDevLauncherController.launcher = launcher
           callback.onSuccess(object : UpdatesInterface.Update {
-            override fun getManifest(): JSONObject {
-              return launcher.launchedUpdate!!.manifest
-            }
-
-            override fun getLaunchAssetPath(): String {
-              return launcher.launchAssetFile!!
-            }
+            override val manifest: JSONObject = launcher.launchedUpdate!!.manifest
+            override val launchAssetPath: String = launcher.launchAssetFile!!
```

Comment:
```suggestion
            override val manifest: JSONObject
            	get() = launcher.launchedUpdate!!.manifest
            override val launchAssetPath: String 
            	get() = launcher.launchAssetFile!!
```
If those values can be changed (and it seems like the final implementation is getting the manifest from that launcher, so it can be changed), we should use a function getter.